src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -179,6 +183,18 @@ class TextGenerationOutput(BaseModel):
         "prompt provided. If streamng is enabled, the next generated token is returned."
         "Otherwise, the full generated sequence is returned."
     )
+    input_tokens: Optional[
```

Comment:
do we want to add additional attributes to the text_generation output? cc @markurtz  @bfineran

Reply:
bumping this @markurtz @bfineran